hack/operatorhub/config.yaml

```diff
@@ -1,4 +1,4 @@
-newVersion: 2.11.0
+newVersion: 2.11.1
 prevVersion: 2.10.0
```

Comment:
I double checked some old tags (2.6.1 for example) to ensure this also was not incremented to 2.11.0.

Reply:
```suggestion
prevVersion: 2.11.0
```

> I double checked some old tags (2.6.1 for example) to ensure this also was not incremented to 2.11.0.

Is it not because `2.6.0` was never released in the operator hub?  See this PR for example https://github.com/elastic/cloud-on-k8s/pull/6322/files.

<img width="156" alt="image" src="https://github.com/elastic/cloud-on-k8s/assets/976373/bff21ae1-71e6-4c96-af04-306d0d1f7e50">